ngx-fudis/projects/ngx-fudis/src/lib/components/divider/divider.stories.ts

```diff
@@ -0,0 +1,20 @@
+import { StoryFn, Meta } from '@storybook/angular';
+import { DividerComponent } from './divider.component';
+import docs from './divider.docs.mdx';
+
+export default {
+  title: 'Components/Divider',
+  component: DividerComponent,
+  parameters: {
+    docs: {
+      page: docs,
+    },
+  },
+} as Meta;
+
+const html = String.raw;
+
+export const Example: StoryFn<DividerComponent> = (args: DividerComponent) => ({
+  ...args,
+  template: html`<fudis-divider />`,
```

Comment:
I would see this more useful if it had an example use case rather than just plain divider.

Simply just: first paragraph - divider - second paragraph